gendiff/scripts/gendiff.py

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env python3
 import argparse
+import json
+
+
+def generate_diff(first_file: str, second_file: str):
+    f1 = json.load(open(first_file, "r", encoding="utf8"))
+    f2 = json.load(open(second_file, "r", encoding="utf8"))
+    keys1 = list(f1.keys())
+    keys2 = list(f2.keys())
+    sorted_union = sorted(list(set(keys1 + keys2)))
+    diff = {}
+    for b in sorted_union:
+        if b in keys2 and b in keys1:
+            if f2[b] == f1[b]:
+                equal = "  " + b
+                diff[equal] = f2[b]
+            else:
+                diff_f1 = "- " + b
+                diff[diff_f1] = f1[b]
+                diff_f2 = "+ " + b
+                diff[diff_f2] = f2[b]
+        elif b in keys1:
+            removed = "- " + b
+            diff[removed] = f1[b]
+        else:
+            new = "+ " + b
+            diff[new] = f2[b]
+
+    jsonstr = json.dumps(diff, ensure_ascii=False, indent=4)
+    return jsonstr
 
 
```

Comment:
hexlet asked you to include two example files into your project, consider storing them in `tests/resources` - it will be easier to run `gendiff` locally